frontend/src/theme/primitives/typography.ts

```diff
@@ -28,18 +34,16 @@ export const genericFonts = {
 }
 
 export const fontSizes = {
-  twoXS: "0.375rem",
-  halfSmDefault: "0.4rem",
-  twoThirdSmDefault: "0.533333rem",
-  xs: "0.5rem",
-  sm: "0.75rem",
-  smDefault: "0.8rem",
+  sm: "0.823rem",
   md: "1rem",
   lg: "1.25rem",
   xl: "1.5rem",
   twoXL: "1.75rem",
   threeXL: "2.25rem",
   fourXL: "3.125rem",
+
+  smNoUnits: 14, // 0.823rem with $font-size-base=17px
```

Comment:
Really it's 13.991, right? I'm a little nervous at all this complex number of pixels. They present off rounding values that could make weird appearances.